File: python/3_231A_ATeam.py
def main():
    n = int(input())
    ans = 00
    for i in range(n):
        sum = 0
        line = input()
        line = line.split()
        for j in line:
            sum += int(j)
        if sum > 1:
            ans += 1
    print(ans)

if __name__ == "__main__":
    main()

File: cpp/3_231A_ATeam.cpp
#include<bits/stdc++.h>

int main(){
    int n, ans = 0;
    std::cin>>n;
    for(int i = 0; i<n; i++){
        int sum = 0, k;
        for(int j  = 0; j < 3; j++){
            std::cin>>k;
            sum += k;
        }
        if (sum > 1){
            ans += 1;
        }
    }
    std::cout<<ans;
    return 0;
}